pkg/kinflate/resource/appresource.go

```diff
@@ -39,3 +38,16 @@ func NewFromPath(path string, loader loader.Loader) ([]*Resource, error) {
 	}
 	return res, nil
 }
+
+//  NewFromPaths returns a Resource slice given a resource path slice from manifest file.
+func NewFromPaths(loader loader.Loader, pathList []string) ([]*Resource, error) {
```

Comment:
Why do you use `[]string`, rather than `...string`?

Reply:
When we call this function from the top level, we always have `[]string`. So we only export this function and make the argument as a slice of string.

Reply:
Yeah it doesn't really matter. It's still easy to call though from the top level `NewFromPath(a.loader, a.manifest.resources...)`. But yes, it doesn't matter.